samples/sample_decode/src/pipeline_decode.cpp

```diff
@@ -1559,17 +1558,7 @@ mfxStatus CDecodingPipeline::DeliverLoop(void)
         msdk_atomic_inc32(&m_output_count);
         m_pDeliveredEvent->Signal();
     }
-    return res;
-}
-
-unsigned int MFX_STDCALL CDecodingPipeline::DeliverThreadFunc(void* ctx)
-{
-    CDecodingPipeline* pipeline = (CDecodingPipeline*)ctx;
-
-    mfxStatus sts;
-    sts = pipeline->DeliverLoop();
-
-    return 0;
+    return;
```

Comment:
minor. 'return' is useless here - drop.

Reply:
will do in next changes, I plan to get rid of of semaphores here